doc/how-to/replication/repl_leader_elect.rst

```diff
@@ -44,13 +44,11 @@ Configuration
 *   ``election_fencing_enabled`` -- switches the :ref:`leader fencing mode <repl_leader_elect_fencing>` on and off.
     For the details, refer to the :ref:`option description <cfg_replication-election_fencing_enabled>` in the configuration reference.
 
-Besides, it is important to know that
-being a leader is not the only requirement for a node to be writable.
-A leader node should have its :ref:`read_only <cfg_basic-read_only>` option set
-to ``false`` (``box.cfg{read_only = false}``),
-and its :ref:`connectivity quorum <cfg_replication-replication_connect_quorum>`
-should be satisfied (``box.cfg{replication_connect_quorum = <count>}``)
-or disabled (``box.cfg{replication_connect_quorum = 0}``).
+Note that a leader should also satisfy the following requirements:
+
+*   The :ref:`read_only <cfg_basic-read_only>` option is to ``false``.
+
+*   The number of connected nodes in a replica set should be more than a half (see :ref:`bootstrap_strategy <cfg_replication-bootstrap_strategy>`).
```

Comment:
I think it's better to say there that the leader shouldn't be in "orphan" state.

Reply:
I meant to replace this line with "The leader shouldn't be in the orphan state".
"Not being an orphan" means the same thing as "there are enough connected nodes".

You can leave either the "The leader shouldn't be in the orphan state" line, or the "The number of connected nodes ..." line. But if you choose the second variant, it'll be hard to describe all the corner cases. A majority of connections is only important during bootstrap. During reconfiguration or recovery the majority is not required.

Reply:
ooh, I seem thanks!